src/App.tsx

```diff
@@ -10,33 +11,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabs, setTabs] = useState(tabs[0]);
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```
Only one tab could be selected